app/routes/sprint-4/managerManageRoutes.js

```diff
@@ -103,6 +104,7 @@ router.get(
         interventionId,
         serviceUser,
         intervention,
+        referral,
```

Comment:
This means we can probably stop passing the smaller parts of the referral in general e.g. `serviceUser` (and just acess them on the `referral` object in the view, but it's a bit of a larger piece of work and time is short so maybe we can not worry about it for now?

Reply:
Good point for sure - I am however a little hesitant about:

i. breaking something
ii. spending more time than is necessary on this, given we'll be throwing it away in two sprints' time and we don't have to live with these decisions forever

So I think I'll leave it.